docs/Users/Install.rst

```diff
@@ -6,7 +6,7 @@ platforms are Linux and Windows. *coala* is known to work on OS X as well.
 *coala* is tested against CPython 3.3, 3.4 and 3.5.
 
 In order to run *coala* you need to install Python. It is recommended,
-that you install Python3 >= 3.3 from http://www.python.org.
+that you install Python3 >= 3.3 from `http://www.python.org/downloads <https://www.python.org/downloads/>`_.
```

Comment:
I think the link name should be like `... you install Python3 >= 3.3 from here` where `here` is a link to the downloads page. But if we're keeping to the current style, make the `http` in the link title as `https`.